libs/designsystem/src/lib/scss/print/components/_ion-item.scss

```diff
@@ -0,0 +1,20 @@
+@media print {
```

Comment:
Is this used to target the [`kirby-item`](https://cookbook.kirby.design/home/showcase/item) component? If that is the case consider either adding a comment stating so or rename the file to reflect that it exists to target the item component.